psi4/driver/procrouting/dft/dft_builder.py

```diff
@@ -121,28 +121,46 @@ def get_functional_aliases(functional_dict):
     for alias in functional_aliases:
         functionals[alias] = dict_functionals[functional_name]
 
-    # if the parent functional is already dispersion corrected, skip to next
+    # if the parent functional is already dispersion corrected:
     if "dispersion" in dict_functionals[functional_name]:
         disp = dict_functionals[functional_name]['dispersion']
-        dashcoeff_supplement[disp['type']]['definitions'][functional_name] = disp
-        # this is to "bless" dft/*_functionals dispersion definitions
+        for formal in functional_aliases:
+            # "bless" the original functional dft/*_functionals dispersion definition including aliases
+            dashcoeff_supplement[disp['type']]['definitions'][formal] = disp
+            # generate dispersion aliases for every functional alias
+            for nominal_dispersion_level, resolved_dispersion_level in _dispersion_aliases.items():
+                if resolved_dispersion_level.lower() == disp["type"]:
+                    alias = formal.replace(disp["type"], nominal_dispersion_level.lower())
+                    if alias not in functionals:
+                        dashcoeff_supplement[disp['type']]['definitions'][formal] = disp
+                        functionals[alias] = dict_functionals[functional_name]
         continue
```

Comment:
I don't think this `continue` does anything, does it?

Reply:
Ooops, I see the loop now that I viewed the whole file.  My mistake!

Reply:
It's meant to skip the next if block.